src/Aquarius.Client/Samples/Client/ServiceModel.cs

```diff
@@ -871,7 +949,7 @@ public class GetObservations : IReturn<SearchResultObservation>, IPaginatedReque
         [DataMember(Name = "depthValue")]
         public double? DepthValue { get; set; }
         [DataMember(Name = "detectionCondition")]
-        public DetectionConditionType? DetectionCondition { get; set; }
```

Comment:
This is an example of one of two forms of breaking changes in AQSamples 2020.05 API.

All the changes are related to the detection condition changing from an enumeration of two values to a more general class.

So this:
```C#
public enum DetectionConditionType
{
    NOT_REPORTED,
    NOT_DETECTED
}
```

becomes this:
```C#
public class ResultDetectionCondition
{
    public string Id { get; set; }
    public string CustomId { get; set; }
    public string Name { get; set; }
    public string Description { get; set; }
    public string SystemCode { get; set; } // With string values of "NOT_DETECTED" or "NOT_REPORTED"
    public AuditAttributes AuditAttributes { get; set; }
}
```

## Form one - Request property changes

In the request DTO below, the `DetectionCondition` request property changed its type from a nullable enumeration to a plain string.

### Affect on running code based on the previous SDK: - None

JSON requests from older SDKs will keep working, since enumeration values were already serialized as JSON strings.

### Affect on software after upgrading to SDK v20.3.1: - Compilation error

You can't assign an enumerated value to a string in C#. So you'll need to modify any code accordingly.